httpipfs.go

```diff
@@ -119,6 +130,23 @@ func (hi *HttpIpfs) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 
 	selNode := unixfsnode.UnixFSPathSelectorBuilder(path.String(), dagScope.TerminalSelectorSpec(), false)
 
+	sig := RequestSignature{
+		requestId: req.Header.Get("X-Signature"),
```

Comment:
Can we use `X-Request-Id`? This header is commonly used for distributed tracing, therefore it's more likely to be included in requests from other clients than SPARK.